dev-docs/bidders/dexerto.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: bidder
+title: Dexerto
+description: Prebid dexerto Bidder Adaptor
+pbjs: true
+biddercode: dexerto
+media_types: banner
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+gpp_sids: usstate_all
+schain_supported: false
+safeframes_ok: false
+ortb_blocking_supported: false
+dsa_supported: false
+deals_supported: true
+floors_supported: true
+sidebarType: 1
+---
+
+### Bid Params
+
+| Name          | Scope     | Description           | Example        | Type     |
+|---------------|-----------|-----------------------|----------------|----------|
+| `placement_id`| mandatory | Placement Id          | `110003`       | `number` |
+| `height`      | optional  | Height of the creative| `250`          | `number` |
+| `width`       | optional  | Width of the creative | `300`          | `number` |
+| `domain`      | optional  | Domain                | `'domain.com'` | `string` |
```

Comment:
@pranavsheth please remove `domain`

Reply:
@ncolletti - we have removed the domain as requested from here also.